how-to/integrate-with-workspace/framework/OpenFin.WPF.TestHarness/WorkspaceManagement.cs

```diff
@@ -15,13 +15,13 @@ internal class WorkspaceManagement: ISnapShotProvider<ApplicationSnapshot>
         Dispatcher dispatch;
         WindowDirectory windowDirectory;
 
-        public WorkspaceManagement(Dispatcher dispatcher)
+        public WorkspaceManagement(Dispatcher dispatcher, string workspaceChannelId)
         {
             dispatch = dispatcher;
             windowDirectory = new WindowDirectory();
             NameValueCollection appSettings = ConfigurationManager.AppSettings;
             bool workspaceAutoConnect = bool.Parse(appSettings.Get("workspaceAutoConnect") ?? Settings.DefaultWorkspaceAutoConnect);
-            string workspaceChannelId = appSettings.Get("workspaceChannelId") ?? Settings.DefaultWorkspaceChannelId;
+            //string workspaceChannelId = appSettings.Get("workspaceChannelId") ?? Settings.DefaultWorkspaceChannelId;
```

Comment:
Can we delete this if we are making it mandatory to pass.

Reply:
Removed commented line